styles/focus/catppuccin.user.css

```
@@ -0,0 +1,441 @@
+/* ==UserStyle==
+@name Focus Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/focus
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/focus
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/focus/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Afocus
+@description Soothing pastel theme for Focus SIS
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+@-moz-document regexp('^(https?:\\/\\/)?([a-zA-Z0-9-]+\\.)+focusschoolsoftware\\.com(\\/.*)?$') {
```

```suggestion
@-moz-document domain('focusschoolsoftware.com') {
```
Should have the same behavior if I'm reading this regex correctly.

---

focusschoolsoftware.com is completely different from the SIS and it won't apply there. The regex i put only works on subdomains of it, which is how the SIS is setup by most schools.